docs/guides/sdk/authentication.md

```diff
@@ -0,0 +1,156 @@
+---
+contributors: Kevin Lewis, Tim de Heiden
+description: Learn about all of the ways to control authentication with the Directus SDK
+---
+
+# Directus SDK Authentication
+
+The Directus SDK provides multiple ways to [authenticate your requests](/reference/authentication), by either generating
+an access/refresh tokens through login, or by setting a static token.
+
+## Generate, Use, and Refresh Access Tokens
+
+### Using the `authentication()` Composable
+
+```js
+import { createDirectus, authentication } from '@directus/sdk';
+const client = createDirectus('http://directus.example.com').with(authentication());
+await client.login(email, password);
+await client.refresh();
+await client.logout();
+```
+
+This approach will handle token refreshing automatically. The current token is stored inside of the initialized client.
+
+### Using the `rest()` Composable
+
+```js
+import { createDirectus, rest, login, refresh, logout } from '@directus/sdk';
+const client = createDirectus('http://directus.example.com').with(rest());
+const user = await client.request(login(email, password));
+await client.request(refresh('REFRESH TOKEN'));
+await client.request(logout('REFRESH TOKEN'));
```

Comment:
For `refresh()` the refresh token is the second parameter and not required in `cookie` mode: 
 * `mode` Whether to retrieve the refresh token in the JSON response, or in a httpOnly secure cookie. One of json, cookie.
 * `refresh_token` The refresh token to use. If you have the refresh token in a cookie through /auth/login, you don't have to submit it here.

Same with the token for `logout()`:
* `refresh_token` The refresh token to use. If you have the refresh token in a cookie through /auth/login, you don't have to submit it here.

Reply:
Can I get a bit more info on this one?

Reply:
So the `refresh()` call in this example is wrong as the first parameter is the `mode`, it should be:
```js
await client.request(refresh('REFRESH TOKEN'));
```

Also the statement below should be adjusted, since the refresh token doesn't need to be stored manually & passed to the calls in `cookie` mode, only when using `json` mode.